src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -3826,7 +3847,8 @@ namespace FanCoilUnits {
                              FanCoil(FanCoilNum).CoolCoilLoopSide,
                              FanCoil(FanCoilNum).CoolCoilBranchNum,
                              FanCoil(FanCoilNum).CoolCoilCompNum);
-        Calc4PipeFanCoil(state, FanCoilNum, ZoneNum, FirstHVACIteration, QUnitOutNoHC);
+        // no load output, requires setting eHeatCoilCyclingR = 0.0, for electric heating coils
+        Calc4PipeFanCoil(state, FanCoilNum, ZoneNum, FirstHVACIteration, QUnitOutNoHC, _, 0.0);
```

Comment:
The new argument *eHeatCoilCyclingR* is set to zero to calculate *Fancoilunit* output at zero load condition.